assets/javascripts/discourse/templates/admin-wizards-custom-fields.hbs

```diff
@@ -14,23 +14,27 @@
   opts=messageOpts
   type=messageType
   url=documentationUrl
-  component='custom_fields'}}
+  component="custom_fields"}}
 
 <div class="admin-wizard-container">
   {{#if customFields}}
     <table>
-      <tr>
-        {{#each fieldKeys as |key|}}
-          <th>{{i18n (concat "admin.wizard.custom_field." key ".label")}}</th>
+      <thead>
```

Comment:
I've checked this table and its dropdowns are a too wide and look the same on the master branch. Here's a video of how they look on the master branch.

https://user-images.githubusercontent.com/19965650/114768668-d20a4900-9d86-11eb-8f56-86db6843adbb.mov

Reply:
@fzngagan They're not like this on master, e.g. thepavilion.io is running the latest. Please check again.

<img width="1148" alt="Screen Shot 2021-04-15 at 10 16 18 AM" src="https://user-images.githubusercontent.com/5931623/114796198-b8dbb980-9dd3-11eb-9104-3a04ea43574e.png">

Reply:
@angusmcleod 
I tried to repro with the `hbs-lint` branch again but it looks fine now. Can you check on your side once. Weird.